website/src/contribute/committer-guide.md

```diff
@@ -77,15 +95,25 @@ At some point in the review process, the change to the codebase will be
 complete. However, the pull request may have a collection of review-related
 commits that are not meaningful to preserve in the history. The reviewer should
 give the LGTM and then request that the author of the pull request rebase,
-squash, split, etc, the commits, so that the history is most useful. Favor
-commits that do just one thing. The commit is the smallest unit of easy
+squash, split, etc, the commits, so that the history is most useful:
+* Favor commits that do just one thing. The commit is the smallest unit of easy
 rollback; it is easy to roll back many commits, or a whole pull request, but
 harder to roll back part of a commit.
+* Commit messages should tag JIRAs and be otherwise descriptive.
+It should later not be necessary to find a merge or first PR commit to find out what caused a change.
+* Squash the "Fixup!", "Address comments" type of commits that resulted from review iterations.
+
+While it is preferred that authors squash commits after review is complete,
+there may be situations where is more practical for the committer to handle this
+(such as when the action to be taken is obvious or the author isn't available).
+The committer may use the "Squash and merge" option in Github (or modify the PR commits in other ways).
+The committer is ultimately responsible and we "trust the committer's judgment"!
 
 ## Merging it!
 
 After all the tests pass, there should be a green merge button at the bottom of
-the pull request.  There are multiple choices and you should choose "Merge pull
+the pull request. There are multiple choices. Unless you want to squash commits
+as part of the merge (see above) you should choose "Merge pull
 request" (the default). This preserves the commit history and adds a merge
```

Comment:
`Merge pull request` actually defaults to whatever operation you did last time. Can you remove `(the default)` and replace it with `and ensure "Create a merge commit" is selected from the drop down.`

Reply:
Done!